subtrees/z_quantum_actions/pyproject.toml

```diff
@@ -28,7 +37,10 @@ exclude_lines = [
 
 [tool.pytest.ini_options]
 log_level="INFO"
+<<<<<<< HEAD
```

Comment:
Something went wrong here! Please remove this and double-check whether it's not an issue in other places as well!

Reply:
Is this something VSCode does with merget conflicts? I don't think I've seen this anywhere else.